modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx

```diff
@@ -177,10 +179,18 @@ export default function TranslationAdminSelector({
 	if (Liferay.FeatureFlags['LPS-114700'] && !adminMode) {
 		return (
 			<Picker
+				active={selectorDropdownActive}
 				as={TriggerButton}
 				displayType={displayType}
 				id={selectorId}
 				items={activeLocales}
+				onActiveChange={(active: any) => {
+					if (active) {
+						onTriggerClick();
+					}
+
+					setSelectorDropdownActive(active);
+				}}
```

Comment:
Much nicer now with callback implementation! :sparkles:  Only a minor naming nit here: Can we call it `on*Change` instead of `on*Click`? It might not be click but keyboard input or something else that does the change. Both `Picker` and `ClayDropDown` internally call it `onActiveChange`. `onSelectorActiveChange` maybe?

Reply:
Yess! I think that `onSelectorActiveChange` is more appropriate. Updated!